examples/chef/sample_app_util/test_files/sample_zap_file.yaml

```diff
@@ -0,0 +1,646 @@
+        - name: FeatureMap
+          value: '0'
+        - name: SupportedLocales
+          value: ''
+        commands: []
+      Network Commissioning:
+        attributes:
+        - name: ClusterRevision
+          value: '1'
+        - name: ConnectMaxTimeSeconds
+          value: ''
+        - name: FeatureMap
+          value: '2'
+        - name: InterfaceEnabled
+          value: ''
+        - name: LastConnectErrorValue
+          value: ''
+        - name: LastNetworkID
+          value: ''
+        - name: LastNetworkingStatus
+          value: ''
+        - name: MaxNetworks
+          value: ''
+        - name: Networks
+          value: ''
+        - name: ScanMaxTimeSeconds
+          value: ''
+        commands:
+        - ScanNetworksResponse
+      OTA Software Update Requestor:
+        attributes:
+        - name: ClusterRevision
+          value: '1'
+        - name: DefaultOtaProviders
+          value: '0'
+        - name: FeatureMap
+          value: '0'
+        - name: UpdatePossible
+          value: '1'
+        - name: UpdateState
+          value: '0'
+        - name: UpdateStateProgress
+          value: '0'
+        commands: []
+      Operational Credentials:
+        attributes:
+        - name: ClusterRevision
+          value: '1'
+        - name: CommissionedFabrics
+          value: ''
+        - name: CurrentFabricIndex
+          value: ''
+        - name: Fabrics
+          value: ''
+        - name: FeatureMap
+          value: '0'
+        - name: NOCs
+          value: ''
+        - name: SupportedFabrics
+          value: ''
+        - name: TrustedRootCertificates
+          value: ''
+        commands:
+        - AttestationResponse
+        - CSRResponse
+        - CertificateChainResponse
+        - NOCResponse
+      Software Diagnostics:
+        attributes:
+        - name: ClusterRevision
+          value: '1'
+        - name: CurrentHeapFree
+          value: '0'
+        - name: CurrentHeapHighWatermark
+          value: '0'
+        - name: CurrentHeapUsed
+          value: '0'
+        - name: FeatureMap
+          value: '1'
+        - name: ThreadMetrics
+          value: ''
+        commands: []
+      Switch:
+        attributes:
+        - name: ClusterRevision
+          value: '1'
+        - name: FeatureMap
+          value: '0'
+        commands: []
+      Thread Network Diagnostics:
+        attributes:
+        - name: ActiveNetworkFaultsList
+          value: ''
+        - name: ActiveTimestamp
+          value: '0'
+        - name: AttachAttemptCount
+          value: '0'
+        - name: BetterPartitionAttachAttemptCount
+          value: '0'
+        - name: ChannelMask
+          value: '0'
+        - name: ChildRoleCount
+          value: '0'
+        - name: ClusterRevision
+          value: '1'
+        - name: DataVersion
+          value: ''
+        - name: DetachedRoleCount
+          value: '0'
+        - name: ExtendedPanId
+          value: '0'
```

Comment:
why do these have default values of 0? these seem configurable / updatable at runtime